cli/logger.go

```diff
@@ -22,57 +22,59 @@ func init() {
 	// WARNING: the grpc logger can only be set during init()
 	// https://godoc.org/google.golang.org/grpc/grpclog#SetLogger
 	// TODO: set verbose to false after testing
-	grpclog.SetLogger(Logger{out: NewOutStream(os.Stdout), verbose: true})
+	grpclog.SetLogger(&Logger{out: NewOutStream(os.Stdout), verbose: true})
```

Comment:
Converting to references, and converting all the functions to pointer methods made sense for Console, but I don't see a need here. There's only a need for a single instance, it's immutable once created, and it's a tiny struct. The change actually obscures that intention. See [value/pointer methods](https://golang.org/doc/faq#methods_on_values_or_pointers).